javascript/node/selenium-webdriver/README.md

```diff
@@ -35,7 +35,7 @@ also find the tests for selenium-webdriver informative.
 const {Builder, By, Key, until} = require('selenium-webdriver');
 
 (async function example() {
-  let driver = await new Builder().forBrowser('firefox').build();
+  const driver = new Builder().forBrowser('firefox').build();
```

Comment:
`build()` returns a Promise, therefore the `await` is required.

Reply:
of, got lost in implementation